lib/gds_api/content_store.rb

```diff
@@ -4,12 +4,19 @@
 require_relative 'exceptions'
 
 class GdsApi::ContentStore < GdsApi::Base
+  EXTENDED_TIMEOUT_FOR_AWS = 8
```

Comment:
Looking at the logs for smart-answers, these are the exceptions: `RestClient::Exceptions::OpenTimeout`. 

So we're saying it can take 4+ seconds to even establish a connection to the content store?

Reply:
Hmm...that is a bit suspicious. Sounds like it's worth some more investigation from us.

Reply:
I think @schmie mentioned the medium latency has gone from about 200ms to 420ms.

Reply:
I'm having trouble locating these exceptions - could you point me somewhere?

Reply:
This is the file I got them from: `/var/apps/smartanswers/log/production.api_client.log`. All the requests are logged, but the errors appear with `ERROR` in the line.

Reply:
OpenTimeout suggests we're backing up on serving requests. I notice that we've only got 2 unicorn workers on the AWS content stores, vs 8 on Carrenza. Shall we up it? Also, do you know where to get graphite metrics for AWS apps, as I don't have any actual data to back this theory up with.